packages/blade/src/components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,260 @@
+  title: 'Components/Card',
+  component: Card,
+  args: {
+    surfaceLevel: 2,
+    headerTitle: 'Payment Links',
+    headerSubtitle: 'Share payment link via an email, SMS, messenger, chatbot etc.',
+    footerTitle: 'Built for Developers',
+    footerSubtitle: 'By Developers.',
+    body:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.",
+    footerPrimaryAction: {
+      text: 'Learn More',
+      onClick: () => {
+        console.log('Primary');
+      },
+      isDisabled: false,
+      icon: undefined,
+      accessibilityLabel: undefined,
+      iconPosition: undefined,
+      isLoading: false,
+      type: undefined,
+    },
+    footerSecondaryAction: {
+      text: 'Try Demo',
+      onClick: () => {
+        console.log('Secondary');
+      },
+      isDisabled: false,
+      icon: undefined,
+      accessibilityLabel: undefined,
+      iconPosition: undefined,
+      isLoading: false,
+      type: undefined,
+    },
+  },
+  argTypes: {
+    surfaceLevel: {
+      table: { category: propsCategory.CARD },
+    },
+    headerTitle: {
+      table: { category: propsCategory.CARD_HEADER_LEADING },
+    },
+    headerSubtitle: {
+      table: { category: propsCategory.CARD_HEADER_LEADING },
+    },
+    prefix: {
+      control: {
+        type: 'select',
+      },
+      mapping: iconMap,
+      options: Object.keys(iconMap),
+      defaultValue: 'LinkIcon',
+      table: {
+        category: propsCategory.CARD_HEADER_LEADING,
+      },
+    },
+    suffix: {
+      control: {
+        type: 'number',
+      },
+      defaultValue: 12,
+      table: { category: propsCategory.CARD_HEADER_LEADING },
+    },
+    visual: {
+      control: {
+        type: 'select',
+      },
+      mapping: visual,
+      options: Object.keys(visual),
+      defaultValue: 'Badge',
+      table: {
+        category: propsCategory.CARD_HEADER_LEADING,
+      },
+    },
+    footerTitle: {
+      table: { category: propsCategory.CARD_FOOTER_LEADING },
+    },
+    footerSubtitle: {
+      table: { category: propsCategory.CARD_FOOTER_LEADING },
+    },
+    footerPrimaryAction: {
+      table: { category: propsCategory.CARD_FOOTER_TRAILING },
+    },
+    footerSecondaryAction: {
+      table: { category: propsCategory.CARD_FOOTER_TRAILING },
+    },
+  },
+  parameters: {
+    docs: {
+      page: Page,
+    },
+  },
+} as Meta<StoryControlProps>;
+
+const CardTemplate = ({ ...args }: StoryControlProps): React.ReactElement => {
```

Comment:
Only added 1 story template, since there are multiple combinations of header, footer props don't think it will be feasible to create separate story examples for each combination. 

Thus created a customisable story.